client/src/services/axios_sev/editHours.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
```

Comment:
On a more architectural note, by having this parent folder be called `axios_sev`. You're making quite an explicit coupling to axios. However it could be that you want to axios to another xhr library in the future. Which would cause this folder plus all imports to be changed to the new folder name. 

I think just having this called something like `./services/editHours.service.js` to be sufficient :)